src/Kunstmaan/NodeSearchBundle/Configuration/NodePagesConfiguration.php

```diff
@@ -679,7 +680,8 @@ protected function renderDefaultSearchView(
     protected function addCustomData(HasNodeInterface $page, &$doc)
     {
         $event = new IndexNodeEvent($page, $doc);
-        $this->container->get('event_dispatcher')->dispatch(IndexNodeEvent::EVENT_INDEX_NODE, $event);
+        $eventDispatcher = LegacyEventDispatcherProxy::decorate($this->container->get('event_dispatcher'));
```

Comment:
Op verschillende andere plaatsen controleren we of deze class eerst bestaat. hoeven we dit hier niet te doen?

Reply:
Good catch! Fixed!